tests/kafkatest/services/kafka/kafka.py

```diff
@@ -994,7 +1000,7 @@ def thread_dump(self, node):
     def clean_node(self, node):
         JmxMixin.clean_node(self, node)
         self.security_config.clean_node(node)
-        node.account.kill_java_processes(self.java_class_name(),
+        node.account.kill_process(self.java_class_name(),
```

Comment:
Had to make this change as kafka when brought up with native binary is not a java process.